ocean_provider/utils/util.py

```diff
@@ -198,13 +198,17 @@ def get_compute_result_endpoint():
     return get_config().operator_service_url + "/api/v1/operator/getResult"
 
 
-def get_compute_address():
+def get_compute_info():
     try:
+        limits = dict()
         compute_info = requests.get(get_config().operator_service_url).json()
-        return compute_info.get("address", None)
+        limits["algoTimeLimit"] = compute_info.get("algoTimeLimit", None)
```

Comment:
We have the util for filtering a dictionary https://github.com/oceanprotocol/provider/blob/b49d605b920dd13c92d48cc217a1ccf2929d93b3/ocean_provider/utils/util.py#L308

I think you can add a default to None parameter to it, and reuse it. I can do it if you're too busy.

Reply:
why should we do this?

Reply:
I think it would make for a clearer code. An alternative would be to not declare dict separately and simply do:

```
limits = {
    "algoTimeLimit": compute_info.get("algoTimeLimit"),
    ...
}
```

to avoid declaring the dict() previously. Or simply return this dict value altogether, alongside the address.
Btw. there is no need for the `, None` part, since None is the default fallback.